src/components/NoteItem/NoteItem.scss

```diff
@@ -21,6 +21,13 @@
         background-color: black;
         color: white;
     }
+    &--hide {
```

Comment:
It should be `.container-note__header_img--hide`, not `container-note--hide`, because you not hiding the whole container